workflows/surgical_agent/metadata.json

```diff
@@ -0,0 +1,159 @@
+{
+	"workflow": {
+		"name": "Surgical Agent Workflow with Video Streaming",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "3.5.0",
+			"tested_versions": [
+				"3.8.0"
+			]
+		},
+		"platforms": [
+			"x86_64",
+			"aarch64"
+		],
+		"tags": [
+			"Healthcare AI",
+			"VLM",
+			"Agent",
+			"Video Streaming",
+			"WebRTC",
+			"MONAI"
+		],
+		"ranking": 2,
+		"requirements": {
+			"data": [
+				{
+					"name": "VLM-Surgical-Agent-Framework",
+					"description": "MONAI Vision Language Model Surgical Agent Framework for interactive surgical video analysis",
+					"url": "https://github.com/Project-MONAI/VLM-Surgical-Agent-Framework.git",
+					"branch": "main"
+				}
+			],
+			"libraries": [
+				{
+					"name": "docker",
+					"description": "Docker and Docker Compose for running surgical agent services"
+				},
+				{
+					"name": "git",
+					"description": "Git for cloning the surgical agent framework"
+				}
+			],
+			"system": {
+				"gpu_memory": "10GB+ VRAM recommended for AI models",
+				"ports": [
+					{
+						"port": 8080,
+						"description": "WebRTC video streaming server"
+					},
+					{
+						"port": 8000,
+						"description": "LLM Server (vLLM)"
+					},
+					{
+						"port": 43001,
+						"description": "Whisper ASR service"
+					},
+					{
+						"port": 8050,
+						"description": "Surgical agent Flask application"
+					}
+				]
+			}
+		},
+		"dependencies": {
+			"applications": [
+				"webrtc_video_server"
+			]
+		},
+		"modes": {
+			"all": {
+				"description": "Build and run everything (default)",
+				"run": {
+					"command": "bash <holohub_app_source>/surgical_agent.sh all",
+					"workdir": "holohub_root"
+				}
+			},
+			"build-video": {
+				"description": "Build video streaming server app only",
+				"run": {
+					"command": "bash <holohub_app_source>/surgical_agent.sh build-video",
+					"workdir": "holohub_root"
+				}
+			},
```

Comment:
When running the workflow, this application is being built, right?
Is there any situation that the user need to run this alone?
which means running only `./holohub build webrtc_video_server` for this workflow?